nemo/collections/nlp/modules/common/megatron/megatron_bert.py

```diff
@@ -150,7 +150,7 @@ def restore_weights(self, restore_path: str):
                 model_parallel_rank = torch.distributed.get_rank(group=get_model_parallel_group())
                 mp_restore_path = f'{restore_path}/mp_rank_{model_parallel_rank:02d}/model_optim_rng.pt'
                 logging.info(f'Restoring model parallel checkpoint from: {mp_restore_path}')
-                state_dict = torch.load(mp_restore_path)
+                state_dict = torch.load(mp_restore_path, map_location=lambda storage, loc: storage)
```

Comment:
Could you change this to
```
state_dict = torch.load(restore_path, map_location='cpu')
```
This is how megatron-lm has it: https://github.com/NVIDIA/Megatron-LM/blob/79888e162425e8d64043a9597ee14751bd4b53d1/megatron/checkpointing.py#L187